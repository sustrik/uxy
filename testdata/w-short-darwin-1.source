 6:00  up 1 min, 2 users, load averages: 11.99 4.09 1.57
USER     TTY      FROM              LOGIN@  IDLE WHAT
travis   console  -                 5:59       - -
travis   s000     10.182.0.83       5:59       - nc 127.0.0.1 15782
