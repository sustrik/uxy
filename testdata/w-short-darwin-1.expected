USER     TTY      FROM              LOGIN@  IDLE WHAT 
travis   console  -                 5:59    -    -   
travis   s000     10.182.0.83       5:59    -    "nc 127.0.0.1 15782" 
